src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -60,10 +62,10 @@ public KeyInfoX509Data(X509Certificate cert, X509IncludeOption includeOption)
                     // Can't honor the option if we only have a partial chain.
                     if ((chain.ChainStatus.Length > 0) &&
                         ((chain.ChainStatus[0].Status & X509ChainStatusFlags.PartialChain) == X509ChainStatusFlags.PartialChain))
-                        throw new CryptographicException(CAPI.CERT_E_CHAINING);
+                    throw new CryptographicException(SR.Cryptography_Partial_Chain);
```

Comment:
It turns out VS Code keeps auto-formatting this when I change it, which explains why it happened (I was on a Mac at the time). I'll stick to using full VS to make these changes for now.